src/vs/vscode.d.ts

```diff
@@ -8926,6 +8926,19 @@ declare module 'vscode' {
 		customRequest(command: string, args?: any): Thenable<any>;
 	}
 
+	/**
+	 * Options for [starting a debug session](#debug.startDebugging).
+	 */
+	export interface DebugSessionOptions {
+
+		/**
+		 * Controls whether this session should have a separate repl console or share it
+		 * with the parent session. Has no effect for sessions which do not have a parent session.
+		 * Defaults to "separate".
+		 */
+		repl?: 'separate' | 'mergeToParent';
```

Comment:
Throughout the API we use enums, not string or-types.

Reply:
for reference: https://github.com/microsoft/vscode/wiki/Extension-API-guidelines#enums